win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5880,6 +5894,59 @@ ScaleLong(LONG lValue, PFLOATOBJ pef)
     return lValue;
 }
 
+/* Calculate width of the text. */
+static inline
+BOOL
+APIENTRY
+ftGdiGetTextWidth(
```

Comment:
This function seems to require the freetype lock held, please guard that with an assert :)

Reply:
OK. Done.